### ISSUE TITLE
add_js results in "Incorrect first char in NameObject:"

### ISSUE THREAD

**Issue by mike4420** [py-pdf/PyPDF2#1420]
Replace this: What happened? What were you trying to achieve?

## Environment

Which environment were you using when you encountered the problem?

```bash
$ python -m platform
Linux-5.19.0-kali2-amd64-x86_64-with-Kali-kali-rolling-kali-rolling

$ python -c "import PyPDF2;print(PyPDF2.__version__)"
2.11.1
```

## Code + PDF

This is a minimal, complete example that shows the issue:

```
from PyPDF2 import PdfFileWriter
writer=PdfFileWriter()
writer.addBlankPage(width=200, height=200)
writer.add_js("this.print({bUI:true,bSilent:false,bShrinkToFit:true});")
with open("output.pdf", "wb") as output_stream:
	writer.write(output_stream)
```
```
$ ./genpdf.py 
Incorrect first char in NameObject:((this.print({bUI:true,bSilent:false,bShrinkToFit:true});))
```

**Comment by pubpub-zz** [comment]
The text was wrongly inserted as a NameObject, fixed in PR #1431

**Comment by MartinThoma** [comment]
Fixed by #1439
